src/include/host/topology.h

```diff
@@ -63,7 +63,8 @@
 #define SOF_TKN_COMP_PERIOD_SINK_COUNT          400
 #define SOF_TKN_COMP_PERIOD_SOURCE_COUNT        401
 #define SOF_TKN_COMP_FORMAT                     402
-#define SOF_TKN_COMP_PRELOAD_COUNT              403
+/* this is not used any more, reserved1
```

Comment:
your comment isn't really clear. something like 
"Token retired with ABI 3.2, do not use for new capabilities" would make more sense.

Reply:
Agree. Updated again